mne_connectivity/tests/test_effective.py

```diff
@@ -39,3 +39,40 @@ def test_psi():
 
     assert np.all(conn_cwt.get_data() > 0)
     assert conn_cwt.shape[-1] == n_times
+
+
+def test_psi_time():
+    """Test Phase Slope Index (PSI) estimation across time."""
+    sfreq = 50.0
+    n_signals = 3
+    n_epochs = 10
+    n_times = 500
+    rng = np.random.RandomState(42)
+    data = rng.randn(n_epochs, n_signals, n_times)
+
+    # simulate time shifts
+    for i in range(n_epochs):
+        data[i, 1, 10:] = data[i, 0, :-10]  # signal 0 is ahead
+        data[i, 2, :-10] = data[i, 0, 10:]  # signal 2 is ahead
+
+    # conn = phase_slope_index_time(data, mode="fourier", sfreq=sfreq, freqs=np.arange(4))
+
+    # assert conn.get_data(output="dense")[1, 0, 0] < 0
+    # assert conn.get_data(output="dense")[2, 0, 0] > 0
+
+    # # only compute for a subset of the indices
+    indices = (np.array([0]), np.array([1]))
+    # conn_2 = phase_slope_index_time(data, mode="fourier", sfreq=sfreq, freqs=np.arange(4), indices=indices)
+
+    # # the measure is symmetric (sign flip)
+    # assert_array_almost_equal(
+    #     conn_2.get_data()[0, 0], -conn.get_data(output="dense")[1, 0, 0]
+    # )
+
+    freqs = np.arange(5.0, 20, 0.5)
+    conn_cwt = phase_slope_index_time(
+        data, mode="cwt_morlet", sfreq=sfreq, freqs=freqs, indices=indices
+    )
```

Comment:
Including some of the things commented here would be nice to better reflect the checks happening for the regular PSI function.

So as well as checking that seed -> target connectivity is > 0, should also check that target -> seed connectivity is < 0 and they are identical but just sign-flipped.